### PR TITLE
fix(blade): radio & checkbox icon alignment

### PR DESCRIPTION
**Pull request by anuraghazra**
- Fixed alignment issues in Radio/Checkbox icon
- Fixed SelectorGroupField role

| Before   |      After      |  
|----------|:---------:|
| Screenshot 2022-08-22 at 12 27 57 PM | Screenshot 2022-08-22 at 12 28 41 PM
 |

### CODE REVIEW

File: packages/blade/src/components/Radio/Radio.tsx [hunk 1]
```diff
@@ -73,16 +73,22 @@ const Radio = ({ value, children, helpText, isDisabled }: RadioProps): React.Rea
...
-      <RadioIcon isChecked={state.isChecked} isDisabled={_isDisabled} isNegative={hasError} />
-      <Box>
-        <SelectorTitle>{children}</SelectorTitle>
-        {showHelpText && <SelectorSupportText id={ids?.helpTextId}>{helpText}</SelectorSupportText>}
+      <Box display="flex" flexDirection="column">
+        <Box display="flex" alignItems="center" flexDirection="row">
+          <SelectorInput
+            isChecked={state.isChecked}
+            isDisabled={isDisabled}
+            isNegative={hasError} // TODO: rename to hasError
```
Comment (chaitanyadeorukhkar) [inline packages/blade/src/components/Radio/Radio.tsx:81]: we didn't do this todo?

**Review by chaitanyadeorukhkar — APPROVED** [review]
Minor feedback but rest LGTM 🚀

File: packages/blade/src/components/Form/Selector/SelectorGroupField.tsx [hunk 2]
```diff
@@ -1,28 +1,35 @@
...
   children,
   labelledBy,
   position,
+  accessibilityRole = 'group',
 }: SelectorGroupFieldProps): React.ReactElement => {
   const isReactNative = getPlatformType() === 'react-native';
   let labelPosition: CSSObject['flexDirection'] = position === 'top' ? 'column' : 'row';
   if (isReactNative) labelPosition = 'column';
-
+  const role = accessibilityRole === 'group' && isReactNative ? undefined : accessibilityRole;
```
[the remark this answers is not in the document]
Comment (anuraghazra) [reply packages/blade/src/components/Form/Selector/SelectorGroupField.tsx:23]: We could do that, just want to keep this component a bit generic. And not have a radiogroup role assigned automatically.

File: packages/blade/src/components/Radio/Radio.tsx (hunk 1 above)
Comment (anuraghazra) [reply packages/blade/src/components/Radio/Radio.tsx:81]: Oh yeah forgot it. I'll do it on a different PR as a refactor.

File: packages/blade/src/components/Form/Selector/SelectorGroupField.tsx [hunk 3]
```diff
@@ -1,28 +1,35 @@
...
+  console.log({ role });
```
Comment (divyanshu013) [inline packages/blade/src/components/Form/Selector/SelectorGroupField.tsx:24]: Stray log

**Review by divyanshu013 — APPROVED** [review]
Looks good to me.

File: packages/blade/src/components/Form/Selector/SelectorGroupField.tsx (hunk 3 above)
Comment (anuraghazra) [reply packages/blade/src/components/Form/Selector/SelectorGroupField.tsx]: yeah removed!

**Pull request merged**